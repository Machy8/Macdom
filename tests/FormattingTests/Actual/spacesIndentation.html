div
        div
                div
                        Alice in Wonderland
                        span !
