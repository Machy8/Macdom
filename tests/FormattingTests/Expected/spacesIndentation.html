<div>
        <div>
                <div>
                        Alice in Wonderland
                        <span>
                                !
                        </span>
                </div>
        </div>
</div>
